tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -412,7 +508,19 @@ def test_invalid_message_batch_reference(nhsd_apim_proxy_url, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_invalid_message_reference(nhsd_apim_proxy_url, correlation_id):
     """
-    .. py:function:: Test invalid message reference
+    .. py:function:: Scenario: A user submitting a request with an invalid \
+        message reference receives a 400 'Invalid Value' response
+
```

Comment:
UUID description again

Reply:
Addressed in uuid.rst, linked below given/when/then